mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.kt

```diff
@@ -871,24 +852,24 @@ class MockWebServer : ExternalResource(), Closeable {
     @Throws(IOException::class)
     override fun write(source: Buffer, byteCount: Long) {
       val toRead = min(remainingByteCount, byteCount)
-      if (toRead > 0) {
+      if (toRead > 0L) {
         source.read(buffer, toRead)
       }
       val toSkip = byteCount - toRead
-      if (toSkip > 0) {
+      if (toSkip > 0L) {
         source.skip(toSkip)
       }
       remainingByteCount -= toRead
       receivedByteCount += byteCount
     }
 
     @Throws(IOException::class)
-    override fun flush() = Unit
+    override fun flush() {}
 
     override fun timeout(): Timeout = Timeout.NONE
 
     @Throws(IOException::class)
-    override fun close() = Unit
+    override fun close() {}
```

Comment:
supernit: I like `}` on its own line

Reply:
(though it looks like I need to fixup Http2Connection.Listener.onSettings and Dispatcher.shutdown!)

Reply:
Fixed.